src/08-attestation.adoc

```diff
@@ -153,6 +158,12 @@ Pre-update attestation gives a chance to unbind TDI before runtime update,
 while post-update reattestation can only unbind TDI after runtime update. That
 can remove the gap between update time and attestation time.
 
+> **Note**
+> In this mode, the device will perform updated anyway, no matter the TSM/TVM
+accept the new firmware update or not. If TSM or TVM cannot accept the new
+firmware, the TSM shall terminate the SPDM session with `END_SESSION`,
```

Comment:
Either the TSM terminates the SPDM session or the TVM unbinds from the TDI?

Reply:
Yes, that is better way to keep SPDM session. Just unbind TDI should be enough.